README.md

```diff
@@ -181,9 +181,8 @@ You may also want to check [this article](https://dzone.com/articles/when-http-s
 More on throwing problems: [zalando/problem usage](https://github.com/zalando/problem#usage)
 
 ## Configuration options
-All configuration options are build-time properties, meaning that you cannot override them in the runtime (i.e via environment variables).
 
-- Include MDC properties in the API response (you have to provide those properties to MDC using `MDC.put`)
+- (Build time) Include MDC properties in the API response. You have to provide those properties to MDC using `MDC.put`
```

Comment:
Maybe it's worth to mention that this extension uses ``org.slf4j`` instead of ``jboss logging``. Thus a user has to pick the appropriated MDC ...

Reply:
Good point. On the other hand, quick tests show that setting properties via `org.jboss.logging.MDC` just works, jboss probably propagates everything to slf4j (or is a service provider etc). I'll double check that in real app though.

Reply:
Ah cool, wasn't aware of that. Thanks for pointing this out.